components/enrichers/custom-annotation/component.yaml

```diff
@@ -1,10 +1,14 @@
 name: "custom-annotation"
 description: "Adds a custom annotation enrichment to the passed findings"
 type: "enricher"
+parameters:
+  - name: annotations_key_colon_vals_comma_separated
```

Comment:
This name is a bit overkill imo, I'd rename it to `annotations_key_vals` and document this accordingly